cmd/network.go

```diff
@@ -365,6 +406,17 @@ func (c *networkCmd) updateRequestFromCLI(args []string) (*models.V1NetworkUpdat
 	if viper.IsSet("additional-announcable-cidrs") {
 		additionalCidrs = viper.GetStringSlice("additional-announcable-cidrs")
 	}
+	defaultchildprefixlength := resp.Defaultchildprefixlength
+	if viper.IsSet("default-child-prefixlength") {
```

Comment:
Where is this flag?

Reply:
I think i will remove this altogether, because setting the default-child-prefixlength with metalctl network update is hedious anyway. WDYT ?

Reply:
Removing is okay.